rest-client/src/main/java/org/sagebionetworks/bridge/rest/ApiClientProvider.java

```diff
@@ -227,6 +228,7 @@ public UserSessionInfoProvider getUserSessionInfoProvider() {
         private String email;
         private String password;
         private UserSessionInfo session;
+        private List<UserSessionInfoProvider.UserSessionInfoChangeListener> changeListeners;
```

Comment:
If these are "public" to the rest API, maybe they shouldn't be a nested class? I think of UserSessionInfoProvider as being a "private" class.